--- pyqtbuild/builder.py	2022-06-18 00:37:55.000000000 +1000
+++ pyqtbuild/builder.py	2022-08-19 18:55:22.000000000 +1000
@@ -210,6 +210,8 @@
         pro_lines = []
 
         pro_lines.append('TEMPLATE = subdirs')
+        pro_lines.append('QMAKE_CC = ' + os.environ.get('CC'))
+        pro_lines.append('QMAKE_CXX = ' + os.environ.get('CXX'))
         pro_lines.append('CONFIG += ordered nostrip')
         pro_lines.append('SUBDIRS = {}'.format(' '.join(subdirs)))
 
@@ -359,6 +361,8 @@
                             buildable.target))
 
         pro_lines = ['TEMPLATE = lib']
+        pro_lines.append('QMAKE_CC = ' + os.environ.get('CC'))
+        pro_lines.append('QMAKE_CXX = ' + os.environ.get('CXX'))
 
         pro_lines.append('CONFIG += warn_on')
 

